static/js/factories/resources.js

```diff
@@ -0,0 +1,76 @@
+//@flow
```

Comment:
might be good to call this file `learning_resources` so it's more immediately clear what it's about